nginx/proxy.conf

```diff
@@ -1,3 +1,5 @@
+resolver 127.0.0.1;
```

Comment:
Thanks @rborer for the fix, let me run some sanity check on other repos and the Nginx I have, and get back to you!